CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# 0.2.0
```

Comment:
There's probably some stuff missing from this changelog, I'll go through one more time and check but if you notice anything please let me know!

Reply:
Also, I still need to add the version number in the initial request.

Reply:
Nice, we were really missing a changelog file. I will add one on iOS as well.